playwright.config.ts

```diff
@@ -61,9 +60,8 @@ const config: PlaywrightTestConfig = {
      */
     timeout: 5 * 1000,
   },
-  // ファイルシステムが関連してくるので、Electronテストでは並列化しない
-  fullyParallel: !isElectron,
-  workers: isElectron ? 1 : undefined,
+  fullyParallel: parallel,
+  workers: !parallel ? 1 : undefined,
```

Comment:
二重否定になってるので、逆にした方がわかりやすいかも
```suggestion
  workers: parallel ? undefined : 1,
```